zookeeper-api/src/main/java/org/apache/helix/zookeeper/datamodel/ZNRecord.java

```diff
@@ -49,6 +50,10 @@
   @JsonIgnore(true)
   public static final String LIST_FIELD_BOUND = "listField.bound";
 
+  /** A field name in ZNRecord's boolean fields to enable compression in ZNRecord serializers. */
+  @JsonIgnore
+  public static final String ENABLE_COMPRESSION_BOOLEAN_FIELD = "enableCompression";
```

Comment:
Can we follow the convention that has an enum for the properties?

Reply:
I prefer to do it later as there are other fields in ZNRecord: LIST_FIELD_BOUND.